test/parallel/test-accessor-properties.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+require('../common');
+
+// This tests that the accessor properties do not raise assersions
+// when called with incompatible receivers.
+
+const assert = require('assert');
+
+// Examples are things that calls StreamBase::AddMethods when setting up
+// their prototype
```

Comment:
Since you're here: can you punctuate the comments in this file?

Reply:
:) This comment barely made sense (it was early morning when the last commit came in), hopefully it’s better now.